Modules/Utils/Sources/PocketCastsUtils/Extensions/UI/CustomTextView.swift

```diff
@@ -0,0 +1,77 @@
+#if !os(watchOS)
+import UIKit
+
+/// A custom UITextField that allows a label appearing near the clear button
+public class CustomTextField: UITextField {
+
+    private lazy var commonSpacing: CGFloat = 8
```

Comment:
Maybe I may missing something but do you see this value as something that can be configured later on? Or could this be a constant?

Reply:
I think a lazy var can still be changed after init, you just delaying the initialization to the first use.

For me it can be just a let, pretty sure the compiler will optimize that and just replace whenever it's used.

Reply:
Changed in 2df741f5721cb59e0ad06f116698501ba75cfcfd